GATEWAY/karma.conf.js

```diff
@@ -24,13 +25,21 @@ module.exports = function (config) {
         lcovonly: './coverage/coverage.lcov'
       }
     },
+    htmlReporter: {
```

Comment:
It's what makes the test report.
I added it so that we can get an easy-to-implement report of the test results for the project report 👍